build/build.xml

```diff
@@ -128,13 +128,13 @@
 	<!-- Builds docs for current branch into build/api/output/master -->
 	<target name="docs">
 		<exec dir="."
-			command="phpBB/vendor/bin/sami.php update build/sami-checkout.conf.php"
+			command="php doctum.phar update build/sami-checkout.conf.php"
```

Comment:
I'm sorry but there's Sami leftover again -> `build/sami-checkout.conf.php` is now `build/doctum-checkout.conf.php`, same below with `build/sami-all.conf.php`.